### PR TITLE
chore: Backmerge of release_ewm_3.0

### PR DESCRIPTION
**Pull request by graupnerdemos**
Squashed commit of the following:

commit f5d366ebc8b3a1f1e5311d758da5c3c49fea1fae
Author: graupnerdemos <graupner@demos-deutschland.de>
Date:   Mon Jun 19 17:51:16 2023 +0200

    fix: Allow complete permission evaluation without valid user (#1565)

    We cannot evaluate the permissions via the db without an existing user
    and need to revert to evaluating directly with edt

commit 8b948b3a2a3d0453c130a0cba5955d7f90e3d4e7
Author: graupnerdemos <graupner@demos-deutschland.de>
Date:   Mon Jun 19 16:33:21 2023 +0200

    feat(refs T33234): Add rudimentary support for exposing addon permissions (#1564)

    Addon permissions are not configured in the classic array way the core permissions
    are evaluated in. Thus, all addon permissions, while having been marked as exposed,
    were dropped when building the frontend permission list.

    This change is only a rudimentary solution to exposing the addon permissions as
    the correct way would be to redo the permission checking in the frontend to
    include addon names and eventually use something more akin to hasPermission(name, context)
    with the context being core or an addon name

commit 24e0728b0a10724161f8e7734da93227ec856205
Author: graupnerdemos <graupner@demos-deutschland.de>
Date:   Mon Jun 19 11:57:47 2023 +0200

    fix(refs T33234): Remove incorrect late permission evaluation (#1562)

    The addon hook configurations do not yet provide permission information.
    If we were to evaluate those, we should filter the scripts in the RPC
    method instead of loading all scripts on the users' machine and evaluating
    on the client side.

    In addition, previously no import tabs would have been loadable from addons
    at all if the excel import were disabled.

commit 20a4775ba44d993d47030fd227762dbc81acb486
Author: bakridemos <102021913+bakridemos@users.noreply.github.com>
Date:   Fri Jun 16 15:48:57 2023 +0200

    fix adjust demosplan addon version

    - some changes made in the 0.0.4 are required.
    - adjust composer.lock

    Signed-off-by: Hamza Bakri <bakri@demos-deutschland.de>

commit ba2a3fc0624088822180994f69715bd20b669682
Author: salisdemos <40487461+salisdemos@users.noreply.github.com>
Date:   Wed Jun 14 18:15:33 2023 +0200

    chore: pick sublte button (#1535)

    * feat(refs T21517): Adapt DpButton usage to new dpui version

    demosplan-ui 0.1.3 introduces a new button variant, "subtle".
    As the styles for demosplan-ui are currently implememted
    in demosplan-core, also the new variant is implemented here.
    Since demosplan-ui 0.1.3 drops slots, these places have to be adapted
    to use icons instead of slots. All needed icons are already implemented
    in demosplan-ui.

    When icons come with a proportion signifier class, we
    can further tweak spacing of icons inside buttons. this will
    eventually be released later on.

    (cherry picked from commit d427b942fa62e60f8b549e1462160a6651b58835)

    * chore: Bump demosplan-ui to 0.1.4

    * Update package.json

    ---------

    Co-authored-by: spiess-demos <40452344+spiess-demos@users.noreply.github.com>

commit 03225dcc776ef7966149108504cb717ab4ee6450
Author: graupnerdemos <graupner@demos-deutschland.de>
Date:   Wed Jun 14 14:36:18 2023 +0200

    fix (refs T33142): uses parent statement submit date (#1530)

    If the entity is a segment, it now uses the submit date from the parent statement and not its own date.

    Co-authored-by: AlexMenschoid <meyer-schlichtmann@demos-deutschland.de>

commit 3f2a89f47b683a1d7e71880df025b56ca50c0c87
Author: salisdemos <40487461+salisdemos@users.noreply.github.com>
Date:   Wed Jun 14 13:05:33 2023 +0200

    fix/refs T33256): Break segments overflow (#1528)

    Add overflow-word-break to segments text boxes

commit fd7a721f5161e85df13480f0530a25bbcb156491
Author: Steffi <101879352+gruenbergerdemos@users.noreply.github.com>
Date:   Wed Jun 14 11:17:50 2023 +0200

    feat(refs T33153): Revert custom label for multiselect (#1505)

    * feat(refs T33153): Revert custom label for multiselect

    Vue multiselect uses here option and not props.

commit ad98d0328cdbdb5924f38615ef436c0153124975
Author: spiess-demos <40452344+spiess-demos@users.noreply.github.com>
Date:   Wed Jun 14 10:57:52 2023 +0200

    feat(refs T33153): Bump demosplan-ui plus some cherrypicks (#1524)

    bump demosplan-ui
    and minor fix: remove fullscreen margin + button padding (#1507)

commit 4f550faca6f9c8f586a9f04e6bfc54c5615e8830
Author: Alexander Marten <marten@demos-deutschland.de>
Date:   Tue Jun 13 16:37:57 2023 +0200

    fix (refs T33109): Flush added similar statement submitter on create new statement.

commit ebf7903824f504f3c4c862537a2d81e5bcb4da31
Author: AlexMenschoid <meyer-schlichtmann@demos-deutschland.de>
Date:   Thu Jun 8 15:43:47 2023 +0200

    fix (refs T33142): uses parent statement submit date

    If the entity is a segment, it now uses the submit date from the parent statement and not its own date.

commit 281841ae9e0c9fbce0c2844da62999451b29fdb3
Author: Demos-CI <devops-system+github@demos-deutschland.de>
Date:   Tue Jun 13 11:18:42 2023 +0000

    style: Apply php-cs-fixer

commit b6d761eeb5448e8d52baa0c3b7015973c304184f
Author: Alexander Marten <marten@demos-deutschland.de>
Date:   Tue Jun 13 13:13:24 2023 +0200

    fix (refs T33142): Add submitDate and authoredDate to export of statements/segments if given.

    - only add rows in case of content is given.
    - remove unused translation
    - rename variables to enhance readability

commit f7e5a282954163e672e8e305b9a8cefe9e2b1015
Author: Alexander Marten <marten@demos-deutschland.de>
Date:   Mon Jun 12 15:06:35 2023 +0200

    fix (refs T33144): Avoid interpreting a 0 as timestamp.

    Actually authored date can be null/empty.
    Export this that way by ensure getting string date in both cases by already existing method
    and make another error prone converting obsolete.

### PR Checklist

Delete the checkbox if it doesn't apply/isn't necessary.

- [ ] Tests updated/created
- [ ] Update documentation
- [ ] Link all relevant tickets
- [ ] Move the tickets on the board accordingly

### CODE REVIEW

File: yarn.lock [hunk 1]
```diff
@@ -1438,9 +1438,9 @@
   integrity sha512-jwx+WCqszn53YHOfvFMJJRd/B2GqkCBt+1MJSG6o5/s8+ytHMvDZXsJgUEWLk12UnLd7HYKac4BYU5i/Ron1Cw==
 
 "@demos-europe/demosplan-ui@^0":
-  version "0.1.6"
-  resolved "https://registry.yarnpkg.com/@demos-europe/demosplan-ui/-/demosplan-ui-0.1.6.tgz#90708b4c754fdc9366f563e2cc07d0fd5fc41148"
-  integrity sha512-kq8Jkk+r1yfGoQ3aIUCjolhHtne5p+NGkaNjcrg1Xtm58AuDb6on/CcYomMeXJGeOcRQZ0P4N7SluBP888pFlA==
+  version "0.1.4"
```
Comment (spiess-demos) [inline yarn.lock:1441]: I think this change can be dropped.

**Review by spiess-demos — CHANGES_REQUESTED** [review]
one concern only from a FE perspective.

**Review by mussbach — APPROVED** [review]
looks like there are only sensible changes in this merge PR. Code review only

File: yarn.lock (hunk 1 above)
Comment (graupnerdemos) [reply yarn.lock:1441]: hm. I'd rather just update demosplan-ui afterwards.

**Review by spiess-demos — APPROVED** [review]
lgtm (code review only)

**Pull request merged**